packages/components/Shimmer/src/Shimmer.tsx

```diff
@@ -0,0 +1,105 @@
+/** @jsx withSlots */
+import {
+  compose,
+  UseSlots,
+  buildProps,
+  mergeProps,
+  withSlots
+} from '@fluentui-react-native/experimental-framework';
+import { requireNativeComponent } from 'react-native';
+import { IViewProps } from '@fluentui-react-native/adapters';
+import * as React from 'react';
+
+const shimmerName = "Shimmer"
+
+const NativeShimmerView = requireNativeComponent('MSFShimmerView');
+
+export interface IShimmerViewAppearance {
```

Comment:
Can you document the difference between `IShimmerViewAppearance` and `IShimmerAppearance`?

Reply:
Yeah, they're documented inline at the prop level, but I can move the documentation here